src/controllers/user/userStatus.ts

```diff
@@ -0,0 +1,45 @@
+import { type Request, type Response, type NextFunction } from 'express'
+import { userStatusQuery } from '../../queries'
+import { type IUser } from '../../interfaces/models'
+import CustomError from '../../helpers/CustomError'
+import { validateUserId } from '../../validation'
+
+const updateStatus = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const { status }: { status: string } = req.body
+
+    if (status !== 'active' && status !== 'deactive') throw new CustomError(400, 'Validation Error')
```

Comment:
use validation `oneOf`
```suggestion
```

Reply:
`    gender: yup.string().oneOf(['male', 'female']).required('Gender is required'),`